mindsdb/integrations/handlers/ray_serve_handler/ray_serve_handler.py

```diff
@@ -37,9 +39,19 @@ def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[
         args['target'] = target
         self.model_storage.json_set('args', args)
         try:
-            resp = requests.post(args['train_url'],
-                                 json={'df': df.to_json(orient='records'), 'target': target},
-                                 headers={'content-type': 'application/json; format=pandas-records'})
+            print(pd.__version__)
+            if args.get('parquet', False):
+                buffer = io.BytesIO()
+                df.attrs['target'] = target
```

Comment:
we can pass target as parameter too
for example:
```
resp = requests.post(args['train_url'], 
   files={'df': buffer},
   data={'target': target},
   headers=headers
)
```

Reply:
I see target is added to request data. Can we remove it from dataframe columns? Or it still required?
`df.attrs['target'] = target`